core/include/drivers/stm32_bsec.h

```diff
@@ -0,0 +1,135 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (c) 2017-2019, STMicroelectronics
+ */
+
+#ifndef __STM32_BSEC_H
+#define __STM32_BSEC_H
+
+#include <stdint.h>
```

Comment:
With `<types_ext.h>` below this isn't needed, on the other hand it's quite harmless too.